frontend/packages/console-shared/src/components/formik-fields/multi-column-field/multicolumn-field-utils.ts

```diff
@@ -0,0 +1,17 @@
+import * as _ from 'lodash';
+import { gridItemSpanValueShape } from '@patternfly/react-core';
+
+export const getSpans = (totalFieldCount: gridItemSpanValueShape): gridItemSpanValueShape[] => {
+  const spans: gridItemSpanValueShape[] = _.fill(
+    Array(totalFieldCount),
+    Math.trunc(12 / totalFieldCount),
+  ) as gridItemSpanValueShape[];
```

Comment:
Same here, is the type & cast needed by TypeScript?

Reply:
Yes, I was getting the type error because after adding return type ```gridItemSpanValueShape[]``` to this utils.

Reply:
Ah right... because `_.fill` has probably a very generic type (if any). Well, it's a nit at this point, but you don't have to type the variable if you're casting. It's always _that_ type.